dash/common.py

```diff
@@ -96,6 +96,10 @@ def component_git_version():
             datetime.strptime(git_time, "%Y-%m-%d %H:%M:%S %z"),
             when=datetime.now(tz=pytz.timezone("Australia/Sydney")),
         )
+    else:
+        natural_time = (
```

Comment:
Perhaps an error message should be shown if the git command fails instead?

Reply:
"Error Loading Timestamp"?